### ISSUE TITLE
Should we allow more than just strings as module attributes?

### ISSUE THREAD

**Issue by littledan** [tc39/proposal-module-attributes#59]
We've talked about generalizing to static-looking object literals in general. A [more scoped generalization](https://github.com/tc39/proposal-module-attributes/pull/57#discussion_r428039527), proposed by @Jack-Works , would be just allowing Numbers, BigInts, null, booleans, and (maybe?) undefined.

I want to propose that we could go to Stage 2 with just strings, and consider generalizations between Stage 2 and 3. In particular, I believe the generalizations @Jack-Works proposes would not be core to the data model or other fundamental decisions that we need to assess for the viability of the proposal overall.

**Comment by xtuc** [comment]
I agree with it being a stage 3 concern. One thing I would like to avoid is promoting hacks to pass other types by parsing the strings like `"null", "false", ...` in the proposal or make people thing that's something we planned.
